database/migrations/20240212042820_forms.up.sql

```diff
@@ -0,0 +1,65 @@
+BEGIN;
+
+CREATE SCHEMA forms;
+
+CREATE TYPE forms.status_type AS ENUM ('DRAFT', 'PUBLISHED');
+
+CREATE TABLE forms.form (
+    id bigserial PRIMARY KEY,
+    content JSONB NOT NULL,
+    created_by_id INT NOT NULL,
+    owner_id INT NOT NULL,
+    "status" forms.status_type DEFAULT 'DRAFT',
+    created_at timestamp DEFAULT (NOW() AT TIME ZONE 'UTC'),
+    updated_at timestamp DEFAULT NULL
+);
```

Comment:
Do we need updatedBy?

Reply:
where is the reference?

Reply:
- @ivinayakg what @prakashchoudhary07 is referring to is having a column to track who last updated the form, this will be helpful as we're allowing the user to update the form.

Reply:
Sure willl do the changes